recipes/quantumespresso/v7.3.1/gh200/environments.yaml

```diff
@@ -0,0 +1,27 @@
+nvhpc-env:
+  compiler:
+    - toolchain: llvm
+      spec: nvhpc
+    - toolchain: gcc
+      spec: gcc@12
+  mpi:
+    spec: cray-mpich@8.1.29
+    gpu: cuda
+  unify: true
+  specs:
+  - cmake%gcc
+  - fftw%nvhpc
+  - nvpl-blas threads=openmp
+  - nvpl-lapack threads=openmp
+  - libxc%nvhpc~cuda
+  - quantum-espresso@7.3.1 %nvhpc +libxc +cuda +nvtx +scalapack
+  - patchelf%gcc
+  - git%gcc
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=90
+  views:
+    default:
+    develop:
+      exclude: ['quantum-espresso']
```

Comment:
Should these views also exclude `git` and `patchelf`?

Reply:
You could also try adding `git` and `patchelf` as
```yaml
  packages:
    - git
    - patchelf
```

Reply:
there is no patchelf installed on santis.

I will add `git` to packages.